src/main/scala/kinesis/mock/KinesisMockService.scala

```diff
@@ -24,6 +25,21 @@ object KinesisMockService extends IOApp {
           "Logging Cache Config"
         )
         cache <- Cache(cacheConfig)
+        _ <- cacheConfig.initializeStreams.toList.flatten
+          .grouped(5)
+          .toList
+          .flatTraverse(
+            _.parTraverse(s =>
+              for {
+                _ <- logger.info(
+                  s"Initializing stream '${s.streamName}' " +
+                    s"(shardCount=${s.shardCount})"
+                )
+                _ <- cache.createStream(s, context, false)
+                _ <- IO.sleep(cacheConfig.createStreamDuration)
```

Comment:
We don't need this sleep until after all of these are created. Should speed up initialization considerably.